Playbook

```diff
@@ -0,0 +1,67 @@
+- name: Install To Do App on new web server
+  hosts: managed_nodes
+  remote_user: ec2-user
+  environment:
+    
+  vars_files:
+    env_vars.yml
+
+  tasks:
+
+  - name: Install Git
+    ansible.builtin.yum:
+      name: git
+      state: 'latest'
+    become: yes
+
+  - name: Install Python 3
+    ansible.builtin.yum:
+      name: python3
+      state: 'latest'
+
+  - name: Install Poetry
+    ansible.builtin.shell: curl -sSL https://install.python-poetry.org | python3
+    args:
+      chdir: ~/.local/bin/
+      creates: poetry
+
+  - name: Create directory for the To-Do App code
+    ansible.builtin.file:
+      path: /opt/todoapp
+      state: directory
+      owner: ec2-user
+    become: yes
+
+  - name: Get the latest version of the App
+    ansible.builtin.git:
+      repo: 'https://github.com/matjadach/to-do-app.git'
+      dest: /opt/todoapp
+      version: exercise-4
+
+  - name: Go to App directory and install project dependencies using Poetry
+    ansible.builtin.command: ~/.local/bin/poetry install
+    args:
+      chdir: /opt/todoapp
+      creates: .venv
+
+  - name: Create .env file from a template
+    ansible.builtin.template:
+      src: .env.j2
+      dest: /opt/todoapp/.env
+
+  - name: Copy the 'todoapp.service' over to host
+    ansible.builtin.copy:
+      src: todoapp.service
+      dest: /etc/systemd/system/todoapp.service
+
+  - name: Kill the running todoapp.service
+    ansible.builtin.command: systemctl kill todoapp.service
+    become: yes
+
+  - name: Start the App by reloading todoapp.service
+    ansible.builtin.systemd:
+      name: todoapp.service
+      state: reloaded
+    become: yes
```

Comment:
So in regards to that - when I followed the instructions and used the following:
`- name: Reload todoapp.service
    ansible.builtin.systemd:
      daemon_reload: yes
    become: yes`
   
When ran ansible-playbook command, the status of this task is "OK" but the app is not working/being served. I logged in to the host vm and used `sudo systemctl` to see if the todoapp.service is running and it is not, hence I went with the above because it seemed to work.

Reply:
Hmm, it _should_ work but I'm willing to believe something odd was going on, happy with this as a solution anyway!